LogicMonitor.DataSDK.Tests/Api/TestLogs.cs

```diff
@@ -15,33 +15,76 @@ public class TestLogs
    {
        public ApiClient apiClient;
        public Resource resource;
+       public Logs logs,batchlogs;
+       public Dictionary<string, string> resourceIds;
+       public Dictionary<string, string> value;
+       string resourceName;
+       public LogsV1 logsv1;
+       string message;
+       string timestamp;
        [SetUp]
        public void Setup()
         {
-            string AccessID = "Shkjafhdfhjshs";
```

Comment:
Can we change the value of AccesID and AccessKey as "DUMMY_ACCESS_ID" & "DUMMY_ACCESS_KEY"